models/Asset.php

```diff
@@ -1676,6 +1676,35 @@ public function addMetadata($name, $type, $data = null, $language = null)
         return $this;
     }
 
+    /**
+     * @param string $name
+     * @param string|null $language
+     *
+     * @return self
+     */
+    public function removeMetadata(string $name, ?string $language = null)
+    {
+        if ($name) {
+            $tmp = [];
+            $name = str_replace('~', '---', $name);
+            if (!is_array($this->metadata)) {
+                $this->metadata = [];
+            }
+
+            foreach ($this->metadata as $item) {
+                if ($item['name'] === $name && ($language == $item['language'] || $language === '*')) {
```

Comment:
I like the idea. I think the docs should be enriched then accordingly 👍

Reply:
I add it to the docs